src/lib/temple/back/vault.ts

```diff
@@ -182,6 +181,9 @@ export class Vault {
   }
 
   static async generateSyncPayload(password: string) {
+    try {
+      await initialize();
+    } catch {}
```

Comment:
is it ok that we will ignore an error from `initialize()`?
this is what will happen if we will remove `try {} catch {}` and `initialize` will throw `Themis Error`
<img width="394" alt="Screenshot 2022-03-21 at 20 17 30" src="https://user-images.githubusercontent.com/10281464/159338551-0a137f78-13d1-4511-b74f-4f3853997e7a.png">

Reply:
Yes. We do not need to initialize Themis a few times. This error happens when we try to initialize Themis more than 1 time.